### ISSUE TITLE
Import fast_rnnt is Failed

### ISSUE THREAD

**Issue by jeff-lim** [k2-fsa/fast_rnnt#27]
Hi team,

I installed successfully with `pip install fast_rnnt`.
But can't import fast_rnnt (as attached img). 

Here's my environment information.
- Win11 / Python 3.11
- CUDA 11.8 / torch 2.0.1

+ Also I tried install from source, but The `import fast_rnnt` is not work with same error message.

<img width="533" alt="image" src="https://github.com/k2-fsa/fast_rnnt/assets/7908677/7659b66d-dbc3-4273-b7d0-f01480e15131">

**Comment by jeff-lim** [comment]
there're only directories... It is okay....?

<img width="900" alt="image" src="https://github.com/k2-fsa/fast_rnnt/assets/7908677/33a5687e-ae89-4841-a1ed-7bb135783fd6">

**Comment by jeff-lim** [comment]
FYI tree of fast_rnnt, fast_rnnt-1.2.dist-info

**fast_rnnt**
<img width="475" alt="image" src="https://github.com/k2-fsa/fast_rnnt/assets/7908677/58349f89-26dd-4ff6-9089-fef085335119">

**fast_rnnt-1.2.dist-info**
<img width="521" alt="image" src="https://github.com/k2-fsa/fast_rnnt/assets/7908677/6b60ab1e-1941-42e9-bfd5-16368917c8c8">

**Comment by csukuangfj** [comment]
Is there a file containing _fast_rnnt in its name?

**Comment by jeff-lim** [comment]
no.. not at all. Actually I wondering the reason why `from/import _fast_rnnt` in **the __init__.py**, **mutual_information.py**

**__init__.py**  [GitRepo](https://github.com/k2-fsa/fast_rnnt/blob/a801adc3b16a7fb637a0b4e71059012dbb44f905/fast_rnnt/python/fast_rnnt/__init__.py#L1)
`
from _fast_rnnt import with_cuda
`
**mutual_information.py** [GitRepo](https://github.com/k2-fsa/fast_rnnt/blob/a801adc3b16a7fb637a0b4e71059012dbb44f905/fast_rnnt/python/fast_rnnt/mutual_information.py#L19)
`
import _fast_rnnt
`

**Comment by csukuangfj** [comment]
After checking pypi.org, I am afraid you cannot use "pip install fast_rnnt" on Windows at pesent.

**Comment by jeff-lim** [comment]
Oh my..
But, I also faced same issue, when I installed from source(git clone, python setup.py...), too.

Is it known issue to you/team?

+ If I install and run the code on Ubuntu, I will be okay to using fast_rnnt with cuda?

**Comment by csukuangfj** [comment]
> Actually I wondering the reason why from/import _fast_rnnt in the _init_.py, mutual_information.py

I bet you have seen `torch._C`. If you don't know what `_C` is, please google it.

Similarly, we have `_fast_rnnt`, which is a Python extension module, implemented in C++/CUDA.

----

> Is it known issue to you/team?

I just looked at the file [setup.py](https://github.com/k2-fsa/fast_rnnt/blob/master/setup.py) and find that it does not support Windows.

Please use the following commands to build fast_rnnt for windows:
```
git clone https://github.com/k2-fsa/fast_rnnt
cd fast_rnnt
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release ..
cmake --build . --target _fast_rnnt
```

After running the above commands, please find the directory where `_fast_rnnt*.pyd` locates and add the directory to the environment variable `PYTHONPATH`.

Similarly, find the directory where `mutual_information_core.dll` locates and add it to the environment variable `path`.


Finally, add the directory `/path/to/fast_rnnt/fast_rnnt/python` to the environment variable `PYTHONPATH`.

Now you should be able to run `python -c "import fast_rnnt"`.

**Comment by csukuangfj** [comment]
> If I install and run the code on Ubuntu, I will be okay to using fast_rnnt with cuda?

Yes, I believe so.
